src/D2L.Bmx/Okta/OktaSessionStorage.cs

```diff
@@ -7,6 +7,12 @@ internal class OktaSessionStorage {
 
 	internal static void SaveSessions( List<OktaSessionCache> sessions ) {
 
+		if( !ConfigFileExists() ) {
```

Comment:
not a fool proof check but I think this is enough 99% of the time

Reply:
Probably for the jumpboxes we'll have to remove the config files on them right? I don't know the setup process or if it's like new images spin up every time

Reply:
The org-wide jumpboxes are either fresh machines or come with user-specific profiles. So it shouldn't be a problem there.
On our own bastion machines there's no separation of any sort.

Ideally we'd check if the username in the config file exists & matches that of the current bmx run, but I'm ok with not doing that for simplicity. Really people just shouldn't set up bmx config file if they don't own the machine (or at least a profile).